eth/blockwatch/block_watcher.go

```diff
@@ -121,7 +121,7 @@ func (w *Watcher) Watch(ctx context.Context) error {
 			return nil
 		case <-ticker.C:
 			if err := w.pollNextBlock(); err != nil {
-				glog.Errorf("blockwatch.Watcher error encountered: %v", err)
+				glog.V(6).Infof("blockwatch.Watcher error encountered - trying again on next polling interval err=%v", err)
```

Comment:
Should we use 5 to correspond with `common.DEBUG`?

Reply:
Fixed in 32ee65fc